sdk/keyvault/azure-security-keyvault-certificates/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ if(BUILD_SOURCE_DEPS)
     GetFolderList(CERTIFICATES)
     foreach(oneFolder IN LISTS BUILD_FOLDERS)
         message("add folder ${oneFolder}")
-        add_subdirectory(${oneFolder})
+        add_subdirectory(${oneFolder} EXCLUDE_FROM_ALL)

```

Comment:
I think add_subdirectory can add source and headers so we don't need to add the headers folder below.

Reply:
no it does not do it properly

Reply:
the second parameter is the include dir. like:
add_subdirectoctory(src, inc, EXCLUDE_FROM_ALL)

try
add_subdirectory("${oneFolder}/src" "${oneFolder}/inc" EXCLUDE_FROM_ALL)